### PR TITLE
Remove unneeded images 

### PR DESCRIPTION
**Pull request by kylelaffoon**
from Upgrade a Cloud Database instance from MySQL 5.1 to 5.6

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-e3348842bc/how-to/)

File: content/cloud-databases/From Managing Cloud Databases HA instance groups i [hunk 1]
```diff
@@ -0,0 +1,22 @@
+From Managing Cloud Databases HA instance groups in the Cloud Control Panel article
```
Comment (stephamon) [inline content/cloud-databases/From Managing Cloud Databases HA instance groups i]: What is all of this??

File: content/cloud-databases/scheduled-backups-for-cloud-databases.sublime-workspace [hunk 2]
```diff
@@ -0,0 +1,187 @@
+{
```
Comment (stephamon) [inline content/cloud-databases/scheduled-backups-for-cloud-databases.sublime-workspace]: And this too. I'm not sure what this is?

**Comment by kylelaffoon** [comment]
- Relates to #705

**Comment by kylelaffoon** [comment]
@stephamon This is my final change on the Cloud Databases images. But it looks like I have an overlap from earlier changes.

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-2059d5e65a/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-fa51bbbe3c/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-fa51bbbe3c/how-to/).

**Comment by kylelaffoon** [comment]
Closing this PR and trying through Atom.

**Pull request closed without merging**